skbio/alignment/_tabular_msa.py

```diff
@@ -1509,8 +1626,7 @@ def _get_join_index(self, other, how):
 
     def _get_sequence_for_join(self, label):
         if label in self.index:
-            # TODO: use .loc when it is implemented.
```

Comment:
Can you grep for other TODOs under `skbio.alignment` and `skbio.io`? I think I marked a couple places where `loc`/`iloc`/`__getitem__` should be used.